### ISSUE TITLE
Inference result from last Linear Classifier of MIP Model

### ISSUE THREAD

**Issue by fjcamillo** [facebookresearch/CovidPrognosis#10]
Got a question about the MIPModel class. Specifically

```
return self.classifier(image_feats_pooled)
```

On the paper, it was mentioned that the output for MIP was

```
MIP Predictions
ICU24: 0.784
Int24: 0.782
Mor24: 0.965
```
but currently output of `MIPModel(image, times, lens)` would be

```
[0.2244, -0.4123]
```
even if trained with new images.  

1. Is it implied that we would be the one to do the update so that we could get something like `[0.53, 0.47] for 2 classes or [0.30, 0.23, 0.47 ] for 3 classes` example would be by adding nn.Softmax() after nn.Linear()? If not, does that mean we'll just treat the output of nn.Linear as the same as the output you've shared on the paper, where we could expect negatives as classes that won't happen, then positives as classes that may happen?
2. For the MIP prediction results stated on the paper, is it possible if you could share if you encountered results where, ICU24 and INT24 was predicted but Mor24 didn't and how the result looks like? 

Thanks again for creating this.

**Comment by anuroopsriram** [comment]
The outputs should be interpreted as logits, so you can apply a [`torch.sigmoid`](https://pytorch.org/docs/stable/generated/torch.sigmoid.html) to get scores between 0 and 1, which is what we report in the paper.

You would only apply `softmax` if you are doing multi-class classification, which is not the case here.

**Comment by fjcamillo** [comment]
@anuroopsriram  yup, it does answer, and also solve the issue I'm encountering. Thank you. 

Would now be closing this issue. Thanks again
